packages/components/src/ListView/ListView.component.js

```diff
@@ -25,14 +25,16 @@ function ListView(props) {
 		<div className={listviewClasses()}>
 			<HeaderListView {...props} />
 			{props.items.length ? (
-				<ItemsListView {...props} />
+				<Items {...props} />
```

Comment:
Directly call `Items` instead of the wrapper that only spreads the received props